src/encoded/types/antibody_lot.py

```diff
@@ -165,9 +165,10 @@ def lot_reviews(characterizations, targets, request):
         if 'histone' in target['investigated_as']:
             is_histone = True
 
-        organism = target['organism']
+        organism = target.get('organism')
```

Comment:
What if this returns None and gets store in `target_organisms[organism] = target['@id']` below?

Reply:
Actually, I'd like to have a `None` here so that I know it has a synthetic target. But it triggers an error during indexing on local launch. These codes here (including the one below) work but don't really make sense and would be addressed in ENCD-4299. My plan is to have this in before ENCD-4299.